client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -42,6 +42,9 @@
 from .. import lock, DockerImage
 from ..stage_change import _StageChange
 
+from verta.dataset.entities import (
+    _dataset
```

Comment:
I recommend importing the `_dataset_version` module, because that is the more "accurate" one for `DatasetVersion`

```suggestion
    _dataset_version
```